src/routes/players.js

```diff
@@ -57,4 +57,85 @@ playerRouter.post("/", (request, response) => {
         .catch((err) => response.status(400).json("Request Failed"))
 })
 
+/**
```

Comment:
lines 61-82 are the comments to add Swagger documentation at `/apidocs` and the code to get a single player

lines 85-113 are the comments to add Swagger documentation at `/apidocs` and the code to PATCH a single player

lines 116-139 are the comments to add Swagger documentation at `/apidocs` and the code to DELETE a single player